packages/ledgerstate/color_test.go

```diff
@@ -0,0 +1,69 @@
+package ledgerstate
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewColoredBalances(t *testing.T) {
+	coloredBalances := NewColoredBalances(map[Color]uint64{
+		{3}: 100,
+		{0}: 200,
+		{1}: 300,
+		{2}: 400,
+	})
+
+	orderedKeys := make([]Color, 0)
+	orderedBalances := make([]uint64, 0)
+	coloredBalances.ForEach(func(color Color, balance uint64) bool {
+		orderedKeys = append(orderedKeys, color)
+		orderedBalances = append(orderedBalances, balance)
+
+		return true
+	})
+	assert.Equal(t, []Color{{0}, {1}, {2}, {3}}, orderedKeys)
+	assert.Equal(t, []uint64{200, 300, 400, 100}, orderedBalances)
+}
+
+func TestColoredBalancesFromBytes(t *testing.T) {
+	coloredBalances := NewColoredBalances(map[Color]uint64{
+		{3}: 100,
+		{0}: 200,
+		{1}: 300,
+		{2}: 400,
+	})
+	marshaledColoredBalances := coloredBalances.Bytes()
+
+	clonedColoredBalances, consumedBytes, err := ColoredBalancesFromBytes(marshaledColoredBalances)
+	require.NoError(t, err)
+	assert.Equal(t, len(marshaledColoredBalances), consumedBytes)
+	assert.Equal(t, clonedColoredBalances.Size(), coloredBalances.Size())
+	assert.Equal(t, clonedColoredBalances.Bytes(), coloredBalances.Bytes())
+
+	color0Balance, color0Exists := clonedColoredBalances.Get(Color{0})
+	assert.True(t, color0Exists)
+	assert.Equal(t, color0Balance, uint64(200))
```

Comment:
The expected value should be in front.
```suggestion
	assert.Equal(t, uint64(200), color0Balance)
```